docs/DeveloperGuide.adoc

```diff
@@ -10,7 +10,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/CS2103JAN2018-F12-B3/main/tree/master
+:repoURL: https://github.com/KevinChuangCH/main
```

Comment:
I'm unsure as to why this line is changed. Shouldn't the repoURL be correct (i.e. pointing to the organisation repo)?